src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -363,10 +385,13 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
         // Authentication listeners
         list($authListeners, $defaultEntryPoint) = $this->createAuthenticationListeners($container, $id, $firewall, $authenticationProviders, $defaultProvider, $configuredEntryPoint);
 
+        $config->replaceArgument(6, $configuredEntryPoint ?: $defaultEntryPoint);
```

Comment:
I would totally avoid storing this config. Having the listeners config like this is not reliable. It'll only be if each listener were forced to register their own processed config themself as a `FirewallListenerConfig`. And use-cases are probably too marginal.

So, keeping the list of registered listeners name should be enough to me.